R/read-write-methods.R

```diff
@@ -199,7 +199,10 @@ readTree <- function(file=NULL, text=NULL, spcl_slt_nm='Unknown', wndmtrx=FALSE,
   other <- rep(NA, length(ids))
   intnds <- 1:length(ids) %in% prinds
   other[intnds] <- ids[intnds]
-  ids[intnds] <- paste0('n', which(intnds))
+  # overwrite internal node ids only if at least one is malformed
+  if (any(grepl('[^a-zA-Z_0-9]', ids[intnds]))){
+    ids[intnds] <- paste0('n', which(intnds))
+  }
```

Comment:
```suggestion
  } else {
      # if using internal node ids as ids, no need for special slot
      other[intnds] <- NA
  }
```
No need to have a special node slot if the same information is contained in the IDs themsevles. TreeMan objects are memory-heavy enough as it is.